dts/bindings/qspi/st,stm32-qspi.yaml

```diff
@@ -50,3 +50,13 @@ properties:
 
         For example
            dma-names = "tx_rx";
+
+    flash-id:
+      type: int
+      description: |
```

Comment:
required: ?
Also, please consider the use of default

Reply:
I think that the "default" and "required" properties are not needed. This option shall be used when somebody by mistake connect `quadspi_bk2` pins (which are dedicated to connecting second flash in two flash setup) to first flash.

Reply:
Please add `required: false`

Reply:
Ok